lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+class Menu
+
+attr_reader :dishes
+
+def initialize
+  @dishes = {}
+end
+
+def add_dish(name,price)
+  @dishes[name] = price
+  @dishes
+end
+
+def remove_dish(name)
+  @dishes.delete_if{|key,value| key == name}
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)